docs/security/credential-vending.md

```diff
@@ -0,0 +1,175 @@
+| `azure-storage-account-name`        | `gravitino.iceberg-rest.azure-storage-account-name` | The static storage account name used to access ADLS data.           | (none)        | Yes      | 0.8.0-incubating |
+| `azure-storage-account-key`         | `gravitino.iceberg-rest.azure-storage-account-key`  | The static storage account key used to access ADLS data.            | (none)        | Yes      | 0.8.0-incubating |
+| `azure-tenant-id`                   | `gravitino.iceberg-rest.azure-tenant-id`            | Azure Active Directory (AAD) tenant ID.                             | (none)        | Yes      | 0.8.0-incubating |
+| `azure-client-id`                   | `gravitino.iceberg-rest.azure-client-id`            | Azure Active Directory (AAD) client ID used for authentication.     | (none)        | Yes      | 0.8.0-incubating |
+| `azure-client-secret`               | `gravitino.iceberg-rest.azure-client-secret`        | Azure Active Directory (AAD) client secret used for authentication. | (none)        | Yes      | 0.8.0-incubating |
+| `adls-token-expire-in-secs`         | `gravitino.iceberg-rest.adls-token-expire-in-secs`  | The ADLS SAS token expire time in secs.                             | 3600          | No       | 0.8.0-incubating | 
+
+### GCS credentials
+
+#### GCS token credential
+
+A credential with a subscope privileged token, to use GCS token credential, you should create an GCS service account and grant proper privilege to it to access the OSS data.
+
+| Gravitino server catalog properties | Gravitino Iceberg REST server configurations      | Description                           | Default value                       | Required | Since Version    |
+|-------------------------------------|---------------------------------------------------|---------------------------------------|-------------------------------------|----------|------------------|
+| `credential-providers`              | `gravitino.iceberg-rest.credential-providers`     | `gcs-token` for GCS token credential. | (none)                              | Yes      | 0.7.0-incubating |
+| `gcs-credential-file-path`          | `gravitino.iceberg-rest.gcs-credential-file-path` | The location of GCS credential file.  | GCS Application default credential. | No       | 0.7.0-incubating |
+
+Please make sure the credential file is accessible by Gravitino, like using `export GOOGLE_APPLICATION_CREDENTIALS=/xx/application_default_credentials.json` before Gravitino Iceberg REST server is started.
+
+## Custom credentials
+
+Gravitino supports custom credentials, you can implement the `org.apache.gravitino.credential.CredentialProvider` interface to support custom credentials, and place the corresponding jar to the classpath of Iceberg catalog server or hadoop catalog.
+
+## Deployment
+
+Besides set credential related configuration, please downloading Gravitino cloud jar and place it to the classpath of Iceberg catalog server or hadoop catalog.
```

Comment:
This whole page needs a proof reading from an English native reviewer.

```suggestion
Besides setting credentials related configuration, please download Gravitino cloud jar and place it in the classpath of the Iceberg catalog server or Hadoop catalog.
```

Reply:
updated